src/Aquifer.API/Services/Auth0Service.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Aquifer.API.Common;
+using Aquifer.API.Configuration;
+using Aquifer.Common.Clients;
+using Aquifer.Common.Utilities;
+using Auth0.AuthenticationApi;
+using Auth0.AuthenticationApi.Models;
+using Auth0.ManagementApi;
+using Auth0.ManagementApi.Models;
+using Auth0.ManagementApi.Paging;
+
+namespace Aquifer.API.Services;
+
+public interface IAuth0Service
+{
+    Task<string> GetAccessTokenAsync(CancellationToken ct);
```

Comment:
I thought about building the fetching of the access token into the scoped service so that it would happen automatically for callers and they wouldn't have to pass it around but decided that could be a later refactor if desired.